playwright/tests/pageobjects/CodeEditor.ts

```diff
@@ -36,6 +38,16 @@ class CodeEditor {
     await expect(this.code).toBeVisible();
   }
 
+  async openBrowsers() {
+    await this.focus();
+    await this.scriptArea.getByRole('button', { name: 'Browser' }).click();
+    await expect(this.page.locator('.browser-content')).toBeVisible();
+  }
+
+  browser(): Browser {
+    return new Browser(this.page);
+  }

```

Comment:
why not direclty return a browser?
```suggestion
  async openBrowsers() {
    await this.focus();
    await this.scriptArea.getByRole('button', { name: 'Browser' }).click();
    await expect(this.page.locator('.browser-content')).toBeVisible();
    return new Browser(this.page);
  }
```